UITests-1.xctestplan

```diff
@@ -70,6 +70,7 @@
       "skippedTests" : [
         "AddSavedItemTests",
         "AddTagsItemTests",
+        "ArchiveAnItemTests",
```

Comment:
supposed to add here?

Reply:
Those tests were already executed in suite 2 so removed from here